### PR TITLE
docs: Initial documentation using VitePress

### LINKED ISSUE #6: Create documentation
Any work already started regarding the documentation? What you think of Vitepress?

I'm using it for one of my libraries and it very easy to deal with.
- Source code: https://github.com/skarllot/Expressions/tree/main/docs
- Published page: https://fgodoy.me/Expressions/

### PR DESCRIPTION
**Pull request by skarllot**
Initial step to introduce documentation for FxKit using VitePress with content entirely based on README file.

- Closes #6

### CODE REVIEW

**Comment by jeffijoe** [comment]
Just checked it out locally to see it and it looks really good! Thanks for doing this!

Do I need to do anything on my end to set up the GH Pages stuff or does it automatically setup a `taxfyle.github.io/fxkit` URL?

**Comment by skarllot** [comment]
Need to ensure that GitHub pages will be deployed using GitHub Actions
image

**Comment by skarllot** [comment]
I've setup VitePress hostname to `https://taxfyle.github.io/FxKit/`

**Comment by skarllot** [comment]
Added a link to the documentation page on README file

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          cache: npm
+          cache-dependency-path: docs/package-lock.json
+      - name: 🌐 Setup Pages
+        uses: actions/configure-pages@v5
+      - name: 📥 Install dependencies
+        run: npm ci
+      - name: 🏗️ Build with VitePress
+        run: |
+          npm run docs:build
+          touch .vitepress/dist/.nojekyll
```
[the remark this answers is not in the document]
Comment (skarllot) [reply .github/workflows/docs.yml:49]: The `.nojekyll` file? It is to avoid GitHub to parse page files as Jekyll template

**Comment by skarllot** [comment]
I need to make some adjustments because of base URL change

**Review by jeffijoe — APPROVED** [review]

**Comment by skarllot** [comment]
Done, fixed favicons path

**Review by jeffijoe — APPROVED** [review]

**Comment by jeffijoe** [comment]
The [deploy failed](https://github.com/taxfyle/FxKit/actions/runs/9390300304/job/25859814332) - perhaps we should run the build step but not the deploy step as part of PRs?

**Pull request merged**